ui/app/models/gcp/config.js

```diff
@@ -0,0 +1,71 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: BUSL-1.1
+ */
+
+import Model, { attr } from '@ember-data/model';
+import { expandAttributeMeta } from 'vault/utils/field-to-attrs';
+
+export default class GcpConfig extends Model {
+  @attr('string') backend; // dynamic path of secret -- set on response from value passed to queryRecord
+
+  /* GCP creds, mutually exclusive with WIF fields */
```

Comment:
nit: Since this phrasing is the same as below but "GCP" and "WIF" are just switched, when I initially skimmed this and the one below it wasn't clear which was which (I read it as "End of GCP fields"). I think it'd be a little clearer to remove the unrelated fields from the comment. 

i.e. `GCP config fields` and below `WIF config fields` - I think the assumption would be they're mutually exclusive since they're separate